frontend/src/app/index.tsx

```diff
@@ -80,6 +97,18 @@ const App = () => {
       }
       return data.whoami.organization.testingFacility[0];
     };
+    updateFacility(getDefaultFacility());
+    updateFacilities(data.whoami.organization.testingFacility);
+    updateUser({
+      id: data.whoami.id,
+      firstName: data.whoami.firstName,
+      middleName: data.whoami.middleName,
+      lastName: data.whoami.lastName,
+      suffix: data.whoami.suffix,
+    });
+    updateOrganization({
+      name: data.whoami.organization.name,
```

Comment:
We're not setting an `id` here, should we?

Reply:
The only reason for this is to update the org name in the settings page. I assume the org is retrieved in the backend from the authenticated user